src/System.Transactions.Local/src/System/Transactions/DurableEnlistmentState.cs

```diff
@@ -11,128 +11,29 @@ namespace System.Transactions
     internal abstract class DurableEnlistmentState : EnlistmentState
     {
         // Double-checked locking pattern requires volatile for read/write synchronization
-        private static volatile DurableEnlistmentActive s_durableEnlistmentActive;
-        private static volatile DurableEnlistmentAborting s_durableEnlistmentAborting;
-        private static volatile DurableEnlistmentCommitting s_durableEnlistmentCommitting;
-        private static volatile DurableEnlistmentDelegated s_durableEnlistmentDelegated;
-        private static volatile DurableEnlistmentEnded s_durableEnlistmentEnded;
+        private static DurableEnlistmentActive s_durableEnlistmentActive;
```

Comment:
@jimcarley, could you review these System.Transactions.Local changes, whether you want them or want them reverted, etc.? Thanks.